libraries/amqp/include/eosio/amqp/transactional_amqp_publisher.hpp

```diff
@@ -0,0 +1,55 @@
+#include <fc/time.hpp>
+
+#include <string>
+#include <memory>
+#include <vector>
+#include <functional>
+#include <deque>
+
+namespace eosio {
+
+/*
+ * A transactional_amqp_publisher manages a connection to a single AMQP server that publishes messages to an exchange
+ * in a reliable manner: connections to the AMQP server are retried, publishing occurs in a transaction,
```

Comment:
How much sense does it make to retry the connection, when you completely stop rodeos after 15 seconds?  Or asked a different way, why is the timeout so short?  I think it's unlikely that a problem that requires opening a new connection would be resolved that quickly.

Reply:
During testing I noticed that losing connection to rabbitmq happens fairly often (every couple of hours) and is quickly regained. Something more like 30-60 seconds probably would be better to give it some more time to become available.

Reply:
Have you not pushed this up yet, or did you change the wrong timeout?

Reply:
Looks like I accidentally reverted it when reverting another change to the interface.